Build.ps1

```diff
@@ -1,3 +1,3 @@
 Invoke-Build Build
```

Comment:
Thanks. Builds take toooo loooong.

Reply:
What should I change to make this so?

Reply:
@TylerLeonhardt Nevermind - figured it out:

```
param(
    [ValidateSet("Debug", "Release")]
    [string]$Configuration = "Debug",

    [string[]]$ModuleName = @( 
        #"Microsoft.PowerShell.GraphicalTools", 
        "Microsoft.PowerShell.ConsoleGuiTools" ) 
)
```

Although, to do this right, it would be nice to have the `GraphicalTools.build.ps1` take a parameter `[Gui|ConoleGui|Both]` such that `Build.ps1` could do this:

```
Invoke-Build Build ConsoleGui
```

But I'm not savvy enough on PS yet to put the logic into the `param` statement above.

Reply:
Nevermind again - I'm now savvy enough. ;-) see latest commit.

Reply:
And FWIW, my changes do not impact the CI/CD builds. Both modules are still being built there, which seems right minded.